addons/utm/wizard/utm_campaign_merge_views.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- Merge UTM Campaigns  -->
+    <record id="utm_campaign_merge_form" model="ir.ui.view">
+        <field name="name">utm.campaign.merge.form</field>
+        <field name="model">utm.campaign.merge</field>
+        <field name="arch" type="xml">
+            <form string="Merge UTM Campaign">
+                <sheet>
+                    <group id="keep campaign">
```

Comment:
Why the id ?

Reply:
Because I need to override it in marketing.automation, I'll use name instead.